frontend/device.lua

```diff
@@ -14,7 +14,7 @@ local function probeDevice()
         return require("device/kindle/device")
     end
 
-    local kobo_test_stat = lfs.attributes("/bin/kobo_config.sh")
+    local kobo_test_stat = lfs.attributes("/bin/kobo_config.sh") or lfs.attributes("/lib/udev/rules.d/99-kobo.rules")
```

Comment:
I would also switch this to `/usr/bin/hwdetect.sh` to mirror the `getCodeName` usage ;).